pkgs/applications/office/mytetra/default.nix

```
@@ -23,6 +23,9 @@ mkDerivation rec {
 
     substituteInPlace app/src/views/mainWindow/MainWindow.cpp \
       --replace ":/resource/pic/logo.svg" "$out/share/icons/hicolor/48x48/apps/mytetra.png"
+
+    substituteInPlace thirdParty/mimetex/mimetex.c \
```

Link to upstream issue https://github.com/xintrea/mytetra_dev/issues/164

```suggestion
    # https://github.com/xintrea/mytetra_dev/issues/164
    substituteInPlace thirdParty/mimetex/mimetex.c \
```

---

Ah, good idea.